.gitmodules

```
@@ -1,3 +1,4 @@
 [submodule "pulumi"]
 	path = pulumi
 	url = git@github.com:pulumi/pulumi.git
+  branch = master
```

```suggestion
	branch = master
```

Edit: actually this still seems to belong to a different branch.